content/concepts/core-concepts.md

```diff
@@ -2,12 +2,31 @@
 title: Core Concepts
 ---
 
-## Entry, Output, Loaders, Plugins
+## Overview
```

Comment:
So this is purely opinionated, and I'm open to whatever my thoughts were: 

`concepts/index.md`: Precursor and 10,000ft view of terms and things to expect
`concepts/core-concepts.md`: Declarative high level overview of what we feel are the core concepts which link to them individually.
`concepts/entry.md`: Deeper understanding of the entry and the multiple ways it can be used.  (and so on for each core concept). 

I wanted to take this approach to allow the first time user to flow from one document to the next. index => core concepts => entry => output => loaders => plugins.